odxtools/compumethods/texttablecompumethod.py

```diff
@@ -39,8 +39,10 @@ def __is_internal_in_scale(self, internal_value, scale: CompuScale):
         if scale.lower_limit is not None and not scale.lower_limit.complies_to_lower(
                 internal_value):
             return False
-        if scale.upper_limit is not None and not scale.upper_limit.complies_to_upper(
-                internal_value):
+        # If no UPPER-LIMIT is defined
+        # the COMPU-SCALE will be applied only for the value defined in LOWER-LIMIT
+        upper_limit = scale.upper_limit or scale.lower_limit
+        if not upper_limit.complies_to_upper(internal_value):
```

Comment:
@andlaus what I understood from the spec is that if no upper limit is defined for TEXTTABLE, SCALE-LINEAR, and SCALE-RAT-FUNC, then the internal value must exactly match the lower limit